blocks/api/raw-handling/index.js

```diff
@@ -57,8 +57,14 @@ export default function rawHandler( { HTML, plainText = '', mode = 'AUTO' } ) {
 		HTML = converter.makeHtml( plainText );
 	}
 
+	const shortcodesConverted = shortcodeConverter( HTML );
```

Comment:
The morphology (esp. the ending in an adjective) irks me a bit. Perhaps renaming and adding a comment:

```js
// An array of HTML strings and block objects. The blocks replace matched shortcodes.
const pieces = …;
```

An alternative could be the quite awkward `htmlAndBlocks`. :)